tests/e2e/specs/settings/applications-passwords.test.js

```diff
@@ -0,0 +1,110 @@
+import {
+    visitAdminPage,
+    __experimentalRest as rest,
+} from '@wordpress/e2e-test-utils';
+
+async function getResponseForApplicationPassword() {
+    return await rest({
+        method: 'GET',
+        path: '/wp/v2/users/me/application-passwords'
+    });
+}
+
+async function createApplicationPassword(applicationName) {
+    await visitAdminPage('profile.php');
+    await page.waitForSelector('#new_application_password_name');
+    await page.type('#new_application_password_name', applicationName);
+    await page.click('#do_new_application_password');
+
+    await page.waitForSelector('#application-passwords-section .notice');
+}
+
+async function revokeAllApplicationPasswords() {
```

Comment:
Since that it's only being used in a single place. I think we can inline this directly in the test instead of creating a separate function?